CHANGELOG.md

```diff
@@ -24,11 +24,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Admin LTE 2.0 used. Assets files changed.
-- Move ``sonata_wrapper`` block on ```standard_layout.html.twig```
+- Move ``sonata_wrapper`` block on ``standard_layout.html.twig``
```

Comment:
> Just for curiosity, why *.rst needs two backtick?

No specific reason. `rst` is as weird as something can be and it decided to use two backticks to enclose literals. See http://www.sphinx-doc.org/en/stable/rest.html 

> Please not branching is changed. We have 2.x, 3.x and master now

Yes, we need to do some changes related to this. Thanks.